sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.0.0-beta.5 (2021-04-07)
 
+### New Features
+
+- Add Active Directory Federation Service (ADFS) support to `ClientSecretCredential`.
```

Comment:
@antkmsft can you take another look at this PR, specifically identity CL changes?